src/applications/burials/components/NoFormPage.jsx

```diff
@@ -0,0 +1,486 @@
+import React, { useEffect, useState } from 'react';
+import { useSelector } from 'react-redux';
+import { apiRequest } from 'platform/utilities/api';
+import { formatSSN } from 'platform/utilities/ui';
+import { isLoggedIn } from '@department-of-veterans-affairs/platform-user/selectors';
+
+const convertDateFormat = date => {
+  date.replace(/^(\d{4})-(\d{2})-(\d{2})$/, '$2/$3/$1');
+};
+
+const formatPhoneNumber = num =>
+  `(${num.substr(0, 3)}) ${num.substr(3, 3)}-${num.substr(6)}`;
+
+const locationOfDeath = {
+  nursingHome: 'Nursing home under VA contract',
+  vaMedicalCenter: 'VA medical center',
+  stateVeteransHome: 'State Veterans home',
+};
+
+const burialAllowanceRequest = {
+  nonService: 'Non-Service connected death',
+  vaMedicalCenter:
+    'Service-connected death (for a Veteran death related to, or resulting from, a service-connected disability)',
+};
+
+const formatCurrency = num => `$${num.toLocaleString()}`;
+const bytesToKB = bytes => `${Math.round(bytes / 1024)} KB`;
+
+const formatAddress = address => {
+  return `${Object.values(address)
+    .filter(line => line !== undefined)
+    .join('\n')}`;
+};
+
+const relationshipType = {
+  spouse: 'Spouse',
+  child: 'Child',
+  parent: 'Parent',
+  executor: 'Executor/Administrator of estate',
+};
+
+const renderFields = [
+  {
+    title: 'Claimant information',
+    id: 'claimant-information',
+  },
+  {
+    title: 'Deceased Veteran information',
+    id: 'deceased-veteran-information',
+  },
+  {
+    title: 'Military history',
+    id: 'military-history',
+  },
+  {
+    title: 'Benefits selection',
+    id: 'benefits-selection',
+  },
+  {
+    title: 'Additional information',
+    id: 'additional-information',
+  },
+];
+
+const generateData = (type, formData) => {
+  switch (type) {
+    case 'claimant-information':
+      return {
+        'Claimant’s first name': formData.claimantFullName.first
+          ? formData.claimantFullName.first
+          : '',
+        'Claimant’s middle name': formData.claimantFullName?.middle ?? 'None',
```

Comment:
The optional chaining guard on L72 isn't present on L69. Also, I love the use of nullish coalescing vs the ternary where the first path returns the same value compared in the conditional. Feels notably more succinct.